controllers/localvolumediscoveryresult_controller.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	localv1alpha1 "github.com/openshift/local-storage-operator/api/v1alpha1"
+)
+
+// LocalVolumeDiscoveryResultReconciler reconciles a LocalVolumeDiscoveryResult object
+type LocalVolumeDiscoveryResultReconciler struct {
+	client.Client
+	Log    logr.Logger
+	Scheme *runtime.Scheme
+}
+
+//+kubebuilder:rbac:groups=local.storage.openshift.io,resources=localvolumediscoveryresults,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=local.storage.openshift.io,resources=localvolumediscoveryresults/status,verbs=get;update;patch
+//+kubebuilder:rbac:groups=local.storage.openshift.io,resources=localvolumediscoveryresults/finalizers,verbs=update
+
+// Reconcile is part of the main kubernetes reconciliation loop which aims to
+// move the current state of the cluster closer to the desired state.
+// TODO(user): Modify the Reconcile function to compare the state specified by
+// the LocalVolumeDiscoveryResult object against the actual cluster state, and then
+// perform operations to make the cluster state reflect the state specified by
+// the user.
+//
+// For more details, check Reconcile and its Result here:
+// - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.7.2/pkg/reconcile
+func (r *LocalVolumeDiscoveryResultReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	_ = r.Log.WithValues("localvolumediscoveryresult", req.NamespacedName)
+
+	// your logic here
```

Comment:
Is this reconciler necessary? It seems it does not do anything useful

Reply:
removed